src/govuk/components/cookie-banner/template.test.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const axe = require('../../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../../lib/jest-helpers')
+
+const examples = getExamples('cookie-banner')
+
+describe('Cookie Banner', () => {
+  describe('question banner', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('renders a heading', () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const $heading = $('.govuk-cookie-banner__heading')
+      expect($heading.text()).toEqual('Cookies on this government service')
+    })
+
+    it('renders heading as escaped html when passed as text', () => {
+      const $ = render('cookie-banner', examples['heading html as text'])
+
+      const $heading = $('.govuk-cookie-banner__heading')
+      expect($heading.html().trim()).toEqual('Cookies on &lt;span&gt;my service&lt;/span&gt;')
+    })
+
+    it('renders heading html', () => {
+      const $ = render('cookie-banner', examples['heading html'])
+
+      const $heading = $('.govuk-cookie-banner__heading')
+      expect($heading.html().trim()).toEqual('Cookies on <span>my service</span>')
+    })
+
+    it('renders main content text', () => {
+      const $ = render('cookie-banner', examples.default)
+
+      const $content = $('.govuk-cookie-banner__content')
+      expect($content.text()).toEqual('We use analytics cookies to help understand how users use our service.')
+    })
+
+    it('renders main content html', () => {
+      const $ = render('cookie-banner', examples.html)
+
+      const $content = $('.govuk-cookie-banner__content')
+      expect($content.html().trim()).toEqual('<p class="govuk-body">We use cookies in <span>our service</span>.</p>')
+    })
+
+    it('renders classes', () => {
+      const $ = render('cookie-banner', examples.classes)
+
+      const $banner = $('.govuk-cookie-banner .govuk-cookie-banner__message')
+
+      expect($banner.hasClass('app-my-class')).toBeTruthy()
+    })
+
+    it('renders attributes', () => {
+      const $ = render('cookie-banner', examples.attributes)
+
+      const $banner = $('.govuk-cookie-banner .govuk-cookie-banner__message')
+
+      expect($banner.attr('data-attribute')).toEqual('my-value')
+    })
+  })
+
+  describe('role and aria attributes', () => {
+    it('has role=region by default', () => {
```

Comment:
Really minor, but this perhaps implies you can override or remove the role, which you can't. Maybe just 

```suggestion
    it('has a role of region', () => {
```